common/lib/xmodule/xmodule/js/spec/main_requirejs.js

```diff
@@ -35,11 +35,17 @@
         baseUrl: '/base/',
         paths: {
             moment: 'common_static/common/js/vendor/moment-with-locales',
-            'draggabilly': 'common_static/js/vendor/draggabilly',
-            'edx-ui-toolkit': 'common_static/edx-ui-toolkit'
+            draggabilly: 'common_static/js/vendor/draggabilly',
+            'edx-ui-toolkit': 'common_static/edx-ui-toolkit',
+            hls: 'common_static/common/js/vendor/hls'
         },
-        'moment': {
-            exports: 'moment'
+        shim: {
+            moment: {
+                exports: 'moment'
+            },
+            hls: {
+                exports: 'Hls'
```

Comment:
nope. `Hls` is the name which exposed on `window` like `window.Hls`. [Details](http://stackoverflow.com/a/29188269)